x/participationrewards/keeper/rewards_holdings.go

```diff
@@ -52,6 +52,8 @@ func (k Keeper) CalcUserHoldingsAllocations(ctx sdk.Context, zone *icstypes.Zone
 
 	userAllocations := make([]types.UserAllocation, 0)
 	icsRewardsAllocations := make([]types.UserAllocation, 0)
+	icsRewardsBalance := sdk.NewCoins()
+	icsRewardsPerAsset := make(map[string]sdk.Dec, 0)
```

Comment:
did we decide we will use var declarations, going forward? @aljo242  @joe-bowman

Reply:
lint fails with this:

`x/participationrewards/keeper/rewards_holdings.go:53:2  prealloc  Consider pre-allocating `userAllocations``

Reply:
so  for arrays, slices and other data structures preallocation is the best practice otherwise there can be race condition